scpd/lms/static/css/vendor/su-identity.css

```diff
@@ -0,0 +1,224 @@
+#su-header .su-container,
```

Comment:
Why are we changing the indentation throughout this file?

Reply:
cause all of our other CSS/SASS files use 4 space tabs?

Reply:
Hrmm, looks like edX uses 2 spaces..

Reply:
true, should we shrink to 2?

Reply:
Maybe we just leave it unchanged for now.
That will reduce the chance of headaches when we backport to the other themes.

After it's migrated elsewhere and symlinked everywhere,
we can consider re-tabbing it to match our other files (or not).

Reply:
`unchanged` meaning leave at 2 spaces (what it is on `master`).

Reply:
done